vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -2,150 +2,166 @@
 sidebarDepth: 2
 ---
 
-# Bundle Management - ent bundle CLI
-The Entando Bundle CLI extends the functionality of **ent** with a modular bundle management system. Starting with Entando 7.1, `ent bundle` orchestrates the lifecycle of a project,  packaging it into convenient recognizable bundles that can be inserted into any Entando Application. This composable approach takes advantage of a single project descriptor and repository, along with centralized [API management](ent-api.md) and [DB and Keycloak services](ent-svc.md).
+# Bundle Management
 
-This document covers `ent bundle` operations and the steps required to create an Entando Bundle.
+The Entando Bundle CLI extends the functionality of **ent** through a dedicated bundle management system. Starting with Entando 7.1, `ent bundle` orchestrates the lifecycle of a project, packaging it into a convenient and recognizable docker-based bundle that can be inserted into any Entando Application. This composable approach takes advantage of a single project descriptor and repository, along with centralized [API management](ent-api.md) and [DB and Keycloak services](ent-svc.md).
+
+Common `ent bundle` operations and the steps required to create an Entando Bundle are discussed in the following sections:
 
 1. [Initialization](#initialization)
 2. [Build](#build)
 3. [Run](#run)
 4. [Package](#package)
 5. [Publish](#publish)
-6. [Deploy](#ecr-commands)
+6. [Deploy](#deploy)
+
+This document also describes the series of `ent ecr` commands that manage bundle interactions with the [Entando Component Repository](../../docs/compose/local-hub-overview.md) (ECR). These are applicable to both docker-based bundles and their git-based predecessors (< Entando 7.1). Finally, notable commands applicable to creating and deploying git-based bundles only are discussed.
+
+## Entando 7.1 Bundle Development
 
-### Bundle Development Overview
-A single JSON descriptor file works as the manifest for an Entando project to be converted into a reusable bundle. The process starts with the initialization step which
-sets up the structure and scaffolding needed for a new project. Alternatively, the ent tool allows a bundle to be initialized directly from the Entando Hub, speeding up the development process. 
+Beginning with Entando 7.1, the `ent bundle` command and its convenience methods govern the structure and files of an Entando Bundle. This format relies on a single JSON descriptor as the project manifest. A new bundle project can be initialized with the default files and folders or from an existing bundle in an Entando hub. 
```

Comment:
`ent bundle` is by definition a command. the Entando Bundle CLI is a tool that enables bundle management, or the functionality of `ent bundle`. the ent bundle cli was introduced front and center at the top of the page; it is implemented via `ent bundle` and the various subcommands/args

Reply:
by definition, `ent bundle` is a command, and it provides convenience methods
https://en.wikipedia.org/wiki/Convenience_function
![Screenshot from 2022-09-13 11-21-01](https://user-images.githubusercontent.com/93157274/189981527-ae997c96-0168-4adb-9dff-8a34d3abe14e.png)

Reply:
in this context, it is definitely not convenience methods because ent bundle adds all sorts of new methods. We will just have different opinions about this one

Reply:
i don't think it's a matter of opinion.